Sources/WebSocket/WebSocket.swift

```diff
@@ -191,7 +230,8 @@ public final class WebSocket: BasicWorker {
         let data = data.convertToData()
         var buffer = channel.allocator.buffer(capacity: data.count)
         buffer.write(bytes: data)
-        send(WebSocketFrame(fin: true, opcode: opcode, data: buffer), promise: promise)
+        let maskKey: WebSocketMaskingKey? = mode.makeMaskKey()
```

Comment:
Do we need to generate a mask key for each send? That adds quite a bit of overhead to each call. I wonder if NIO has any way of doing this for us automatically, or if we can use a static key per websocket connection.

Reply:
Yes, necessary for each one to be unique and new.